src/Chapter1.hs

```diff
@@ -619,9 +634,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let lastNum = mod (abs(n)) 10
```

Comment:
Yeah, perfect usage of `div` and `mod` in here ✨ 

`lastNum` is actually the same as `lastDigit`, implemented above 😉 

And It is better to write `abs n` insteadof `abs n` in Haskell for readability reasons 🙂